### PR TITLE
Swifting WPNUXHelpBadgeLabel

### PR DESCRIPTION
**Pull request by jleandroperez**
### Details:
Analog to #8579, in this small PR we're porting `WPNUXHelpBadgeLabel` over to Swift!

### To test:
1. Hack [this spot](https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/WordPress/Classes/ViewRelated/NUX/NUXViewControllerBase.swift#L161) and paste the following snippet:

```
helpBadge.isHidden = false
helpBadge.text = "1"
```

2. Just in case, comment [the body of this method](https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/WordPress/Classes/ViewRelated/NUX/NUXViewControllerBase.swift#L100)

3. Fresh install the app, and press the `Login` button

Verify that a red bubble shows up on top of the Help button.

Needs Review: @frosty 
Thaaaanks in advance!!

### CODE REVIEW

File: WordPress/Classes/ViewRelated/NUX/NUXHelpBadgeLabel.swift [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    func commonSetup() {
+        layer.masksToBounds = true
+        layer.cornerRadius = 6.0
+        textAlignment = .center
+        backgroundColor = UIColor(fromHex: 0xdd3d36)
+        textColor = .white
+        font = WPFontManager.systemRegularFont(ofSize: 8.0)
+    }
+
+    override func draw(_ rect: CGRect) {
```
Comment (frosty) [inline WordPress/Classes/ViewRelated/NUX/NUXHelpBadgeLabel.swift]: This should override `drawText(in rect: CGRect)`, not `draw(_ rect: CGRect)`.

Comment (jleandroperez) [reply WordPress/Classes/ViewRelated/NUX/NUXHelpBadgeLabel.swift]: Ahhhhh eagle eyes, nice catch!!!

**Comment by jleandroperez** [comment]
@frosty thanks for the review James!! ready for another peek. Should look as follows!

simulator screen shot - iphone se - 2018-02-07 at 07 54 08

**Comment by jleandroperez** [comment]
Thanks a lot!!

**Pull request merged**